app/pages/organization/stats/components/BarChart/BarChartContainer.jsx

```diff
@@ -0,0 +1,212 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import statsClient from 'panoptes-client/lib/stats-client';
+
+import fillTimeSeries from './helpers/fillTimeSeries';
+import filterTimeSeries from './helpers/filterTimeSeries';
+import getDefaultRange from './helpers/getDefaultRange';
+import BarChartBlock from './BarChartBlock';
+
+class BarChartContainer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      binBy: 'day',
+      rangeMax: undefined,
+      rangeMin: undefined,
+      resourceId: 'all',
+      statDataByResource: new Map()
```

Comment:
`new Map()` is potentially an IE11 issue, although it appears to work through BrowserStack